fraim/workflows/risk_flagger/workflow.py

```diff
@@ -204,5 +210,6 @@ async def chunk_processor(chunk: CodeChunk) -> List[sarif.Result]:
                 self.logger.warning("PR URL and/or approver are missing, skipping Add Reviewer")
 
         self.logger.info(pr_comment)
+        print(pr_comment)
```

Comment:
This is a better example of my question, is the standard now that we log + print information logs?

Reply:
That's open question, in my mind.  Would like to discuss with you and Rudi as part of future logging refactors.

For this PR, that's basically correct though.

Log is logs. Print/stdout is what should show on the terminal after live display exits.  In some cases, we'll want to print the same thing to both.

Reply:
that's right I remember you mentioning that yesterday